UKHomeOffice/sgmr-service
# NMSW-797 Update vessel nationality to use the same country autocomplete pattern as person nationality

### AC / Description of changes

New bug found when creating the data for CBP - vesselNationality must be just the ISO 3 digit code and currently we send the country name.

Quickest fix is to replicate what we do for person nationality as that has the same requirement

### To Test

- go to pleasure craft form
- create a new or edit an existing craft
- go to page 2
- ensure `Does this pleasure craft have a Country of Registration?` is set to `Yes`
- type in the field
- > you should see a list of country options that match what you typed
- select a country
- > you should see the `Country Name (ISO Code)` as what you selected
- submit the form
- > in the network tab you should see that we only pass `ISO Code` in the `vesselNationality` field
- edit the vessel in the browser
- > you should see only the `ISO Code` in the `Country of Registration` field


### Notes

---

**Remember to merge to feature branches not `master`.**
